roomserver/input/events.go

```diff
@@ -212,3 +227,50 @@ func processInviteEvent(
 	succeeded = true
 	return nil
 }
+
+func buildInviteStrippedState(
+	ctx context.Context,
+	db storage.Database,
+	input api.InputInviteEvent,
+) ([]gomatrixserverlib.InviteV2StrippedState, error) {
+	roomNID, err := db.RoomNID(ctx, input.Event.RoomID())
+	if err != nil || roomNID == 0 {
+		return nil, errors.New("room unknown")
```

Comment:
As always, please give context to your errors. Please add the room ID.

Reply:
Done.